okta-hosted-login/src/main/resources/application.yml

```diff
@@ -0,0 +1,13 @@
+micronaut:
+  security:
+    authentication: idtoken 
+    oauth2:
+      clients:
+        okta:
+          client-id: '${OKTA_OAUTH2_CLIENTID}'
+          client-secret: '${OKTA_OAUTH2_CLIENTSECRET}'
+          openid:
+            issuer: '${OKTA_OAUTH2_ISSUER}'
```

Comment:
I see what you did now. Using Micronaut's keys might be better because then you wouldn't even need this file.

Reply:
noted